docs/restful-api/services/token-services.md

```diff
@@ -235,6 +235,9 @@ Content-Type: application/x-www-form-urlencoded</p></td>
 - <i>tokenType=APIKEY</i> - specifies the scope of validity of the APIKey; the only licensee with the given number can be validated using this APIKey<br />
 - <i>tokenType=ACTION</i> - identifies licensee for whom action token will be created
 </p>
+<p>expirationTime (optional) - Specify a custom expiration time for the token. If not provided, the default expiration time of 30 minutes will be applied.</p>
+<p>bundleNumber (optional) - Specify the bundle number to display on the Shop checkout page.</p>
+<p>bundlePrice (optional) - Specify the customer's individual bundle price. The default bundle price will be used if no bundlePrice is set.</p>
```

Comment:
Permanent shop token props are missing

Reply:
Changed with the commit https://github.com/Labs64/wiki/pull/15/commits/34a0c025fe428899d9c7352788664027868d1a88